js/controllers/GameCtrl.js

```diff
@@ -28,105 +28,172 @@ angular.module(ctrl_name, []).controller(ctrl_name, ['$scope', '$http', '$locati
-              closeOnConfirm: false
-          }, function () {
-              $http({
-                  method: 'POST',
-                  url: '.delete_game.php',
-                  data: 'session_id=' + AuthService.getSessionId() + '&game_id=' + $routeParams.gameId,
-                  headers: {'Content-Type': 'application/x-www-form-urlencoded'}
-              }).then(function (response) {
-                  sweetAlert("Deleted!", "The game " + $scope.game.description + " has been deleted.", "success");
-                  $location.url('/games-events');
-              });
-          });
-      }
-
-      $scope.editGame = function () {
-        if(!$scope.admin) {
+        if (!$scope.admin) {
+            return;
+        }
+        sweetAlert({
+            title: "Are you sure?",
+            text: "This action will permanently delete the game " + $scope.game.description + ". Do you want to proceed?",
+            type: "warning",
+            showCancelButton: true,
+            confirmButtonColor: "#c52d2f",
+            confirmButtonText: "Delete",
+            closeOnConfirm: false
+        }, function () {
+            $http({
+                method: 'POST',
+                url: '.delete_game.php',
+                data: 'session_id=' + AuthService.getSessionId() + '&game_id=' + $routeParams.gameId,
+                headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
+            }).then(function (response) {
+                sweetAlert("Deleted!", "The game " + $scope.game.description + " has been deleted.", "success");
+                $location.url('/games-events');
+            });
+        });
+    }
+
+    $scope.editGame = function () {
+        if (!$scope.admin) {
             return;
         }
-          $location.url('/edit/game/' + $routeParams.gameId);
-      }
+        $location.url('/edit/game/' + $routeParams.gameId);
+    }
+
+    $scope.displayModal = () => {
+        $scope.showModal = true;
+    }
+
+    $scope.sendEmail = () => {
+        $scope.showModal = false;
+        if (!$scope.admin) {
+            return;
+        }
+        let body = `Dear Attendees,\n\nThank you for signing up for ${$scope.game.description} with RPI Ambulance. The event is ${$scope.game.date} at ${$scope.game.start} hours. Please meet at the garage 5 minutes prior to ensure a prompt departure.\n\n`;
+        body += 'Attendance is as follows:\n';
+        $scope.positions.forEach((p) => {
+            if (p.attendees.length != 0) {
+                body += `${p.title}[s]: ${p.attendees[0].ambulance_name}`;
+                for (let i = 1; i < p.attendees.length; i++) {
+                    body += `, ${p.attendees[0].ambulance_name}`;
+                }
+                body += '\n';
+            }
+        });
+        body += `\nAdditional Info:\n${$scope.formData.additionalinfo}`;
+        body += '\n If you have any questions, you can reach out to secondlt@rpiambualnce.com. Thanks!\n\n';
```

Comment:
Or who sent this email (if it wasn't the 2nd lt)

Reply:
Only the 2nd lt should be sending event emails.

Reply:
Good point. I do think we should add the EES info in there if it's a EESed event. They're a good POC

Reply:
Per Car 3. I should just include 2nd lt and he will route them accordingly

Reply:
I never understood why people choose to create more work for themselves, but ok.